packages/common/driver-definitions/src/driverError.ts

```diff
@@ -92,6 +92,19 @@ export enum DriverErrorType {
     fluidInvalidSchema = "fluidInvalidSchema",
 }
 
+/**
+ * Interface describing errors and warnings raised by any driver code.
+ * Not expected to be implemented by a class or an object literal, but rather used in place of
+ * any or unknown in various function signatures that pass errors around.
+ *
+ * "Any" in the interface name is a nod to the fact that errorType has lost its type constraint.
+ * It will be either DriverErrorType or the specific driver's specialized error type enum,
+ * but we can't reference a specific driver's error type enum in this code.
+ */
+export interface IAnyDriverError extends Omit<IDriverErrorBase, "errorType"> {
```

Comment:
I wonder if it should be other way around - we define IAnyDriverError interface, and then IDriverErrorBase extends it and overwrites errorType with more concrete type. It should compile without using "Omit" thing

Reply:
It is same thing whatever we do. They both have same structure except the different type in errortype. So we have to use Omit in DriverErrorBase if we go as you suggested. So, I would leave it as such.